CORE-6303 & CORE-6475 (#656)
corda/corda-docs-portal: content/en/platform/corda/5.0-dev-preview-2/developing/api/api-cipher-suite.md

title: "Cipher-suite"
menu:
  corda-5-dev-preview:
    identifier: corda-5-dev-preview-api-cipher-suite
    parent: corda-5-dev-preview-api
    weight: 3000
section_menu: corda-5-dev-preview
---

**# Overview**

> The `corda-cipher-suite` module as a part of the `Corda Crypto API` consisting of several modules. The module defines code artifacts that are required to implement integrations with HSMs beyond supported by R3. The module is used by Corda Platform.

Reviewer: ```suggestion
The `corda-cipher-suite` module is one of several modules of the `Corda Crypto API`. The module defines code artifacts that are required to implement integrations with HSMs beyond those supported by R3. The module is used by the Corda platform.
```

> The dependencies on Corda Crypto API are shown in the diagram bellow. 

Reviewer: ```suggestion
The dependencies on the Corda Crypto API are shown in the diagram below. 
```
